static/stylesheets/selection.css

```diff
@@ -17,6 +17,14 @@
     border-radius: 5px;
 }
 
+a:visited {
```

Comment:
This needs `a:hover` too. These should also use a class instead of just targetting all anchor tags, e.g. `.selectionBoxes a:hover`.